docs/localization/LOCALIZATION.md

```diff
@@ -419,107 +523,172 @@ class AppModule {}
+the first priority goes to `fallbackLocalesMap` to see if the targeted translation language can be mapped with the configured fallback map.
+The second priority goes to `supportedLocales` to match the targeted translation language to the first closest possible language, the **locale** may be different.
+If none match, it will fall back to the **default language**.
+
+**Let's assume:**
+
+```typescript
+return {
+  // ...
+  supportedLocales: ['en-GB', 'en-US', 'fr-FR', 'ar-AR'],
+  fallbackLocalesMap: {
+    'en-CA': 'en-US',
+    'fr-CA': 'fr-FR',
+    'de-CH': 'ar-AR',
+    'de': 'fr-FR',
+    'it': 'fr-FR',
+    'hi': 'en-GB',
+    'zh': 'en-GB'
+  },
+  fallbackLanguage: 'ar-AR'
+}
+```
+
+**Fallback scenarios:**
+
+* `en-CA` **fallbacks to** `en-US`, as direct mapping available in fallback locales map.
+* `de-CH` **fallbacks to** `ar-AR`, as direct mapping available in fallback locales map.
+* `de-AT` **fallbacks to** `fr-FR`, as language mapping available in fallback locales map. 
+* `zh-CN` **fallbacks to** `en-GB`, as language mapping available in fallback locales map.
+* `en-AU` **fallbacks to** `en-GB`, as fallback locales mapping unavailable, first nearest language available in supported locales.
+* `fr-BE` **fallbacks to** `fr-FR`, as fallback locales mapping unavailable, first nearest language available in supported locales. 
+* `bn-BD` **fallbacks to** `ar-AR`, as it is the default fallback.
+
+## Scenario 2: Fallback based on `supportedLocales`
+
+In case `fallbackLocalesMap` is not provided and the targeted translation language is unavailable in the list of supported locales,
+the targeted translation language will be compared to the supported locales to find the first closest match, the **locale** may be different.
+If none match, it will fall back to the **default language**.
+
+**Let's assume:**
+
+```typescript
+return {
+  // ...
+  supportedLocales: ['en-GB', 'fr-FR', 'fr-CA', 'ar-AR'],
+  fallbackLanguage: 'en-GB'
+}
+```
+
+**Fallback scenarios:**
+
+* `en-US` **fallbacks to** `en-GB`, as en-GB has the same language with a different region.
+* `fr-BE` **fallbacks to** `fr-FR`, as fr-FR is first in the supported locales list.
+* `it-IT` **fallbacks to** `en-GB`, as it is the default fallback.
+
 # Debugging
 
 ## Runtime: toggle translation on and off
 
-In order to be able to more easily identify which key corresponds to a given text, the ``LocalizationService`` exposes a function ``toggleShowKeys()`` that can be called in order to deactivate or reactivate the translation mechanism at **runtime**.
+To make it easier to identify which key corresponds to a given text, the ``LocalizationService`` exposes a function ``toggleShowKeys()`` that can be called to deactivate or reactivate the translation mechanism at **runtime**.
 While deactivated, the ``translate`` **pipe** and **directive** will output the translation keys instead of their resolved values.
 
 > [!IMPORTANT]
```

Comment:
I would say `NOTE` or `WARNING` for this one 

and `IMPORTANT` for the paragraph just below as it has an impact on the performances